src/app/sale/sale.model.ts

```diff
@@ -0,0 +1,9 @@
+import { CakeInterface } from '../cake/cake.model';
+
+export interface SaleInterface {
```

Comment:
As a good practice, you can add the letter I before the name of your interfaces, to identify them versus the classes. Ex. `IBread` or `iBread`.